bot/exts/evergreen/trivia_quiz.py

```diff
@@ -76,6 +258,30 @@ async def quiz_game(self, ctx: commands.Context, category: str = None) -> None:
             await ctx.send(embed=embed)
             return
 
+        topic = self.questions[category]
+        topic_length = len(topic)
+
+        if questions:
+            if questions > topic_length:
+                await ctx.send(
+                    embed=self.make_error_embed(
+                        f"This category only has {topic_length} questions. "
+                        "Please input a lower value!"
+                    )
+                )
+                return
+
+            elif questions < 1:
+                await ctx.send(
+                    embed=self.make_error_embed(
+                        "Please do at least one question."
```

Comment:
'do' doesn't really fit here, I think it should be something more like "You must choose to complete at least one question."

Reply:
lol grammar

Reply:
resolved